### ISSUE TITLE
🐛 Bug Report: 

### ISSUE THREAD

**Issue by oo00spy00oo** [novuhq/novu#2415]
### 📜 Description

**novu-web** failed to start on Kuberntes:

### 👟 Reproduction steps

Via the guide [here](https://github.com/novuhq/novu/tree/next/docker/kubernetes/kustomize)

`kustomize build . | kubectl apply -f - `

### 👍 Expected behavior

`novu-web` with built assets, not running wih `nx`

You can take a look with [nx-container](https://github.com/gperdomor/nx-tools/blob/main/packages/nx-container/README.md)

### 👎 Actual Behavior with Screenshots

```
> novuhq@ start:static:web /usr/src/app
> cross-env nx run-many --target=start:static:build --projects=@novu/web

/bin/sh: git: not found

 >  NX   Running target start:static:build for 1 project(s):

    - @novu/web

 

> nx run @novu/web:"start:static:build"


> @novu/web@0.10.0 start:static:build /usr/src/app/apps/web
> pnpm envsetup:docker && http-server build -p 4200 --proxy http://localhost:4200?


> @novu/web@0.10.0 envsetup:docker /usr/src/app/apps/web
> chmod +x ./env.sh && ./env.sh && mv ./env-config.js ./build/env-config.js

sh: ./env.sh: not found
 ELIFECYCLE  Command failed.
 ELIFECYCLE  Command failed with exit code 1.

 

 >  NX   Running target "start:static:build" failed

```

### 📃 Provide any additional context for the Bug.

Novu version: v0.10.0

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

**Comment by oo00spy00oo** [comment]
> ### 📜 Description
> **novu-web** failed to start on Kuberntes:
> 
> ### 👟 Reproduction steps
> Via the guide [here](https://github.com/novuhq/novu/tree/next/docker/kubernetes/kustomize)
> 
> `kustomize build . | kubectl apply -f - `
> 
> ### 👍 Expected behavior
> `novu-web` with built assets, not running wih `nx`
> 
> You can take a look with [nx-container](https://github.com/gperdomor/nx-tools/blob/main/packages/nx-container/README.md)
> 
> ### 👎 Actual Behavior with Screenshots
> ```
> > novuhq@ start:static:web /usr/src/app
> > cross-env nx run-many --target=start:static:build --projects=@novu/web
> 
> /bin/sh: git: not found
> 
>  >  NX   Running target start:static:build for 1 project(s):
> 
>     - @novu/web
> 
>  
> 
> > nx run @novu/web:"start:static:build"
> 
> 
> > @novu/web@0.10.0 start:static:build /usr/src/app/apps/web
> > pnpm envsetup:docker && http-server build -p 4200 --proxy http://localhost:4200?
> 
> 
> > @novu/web@0.10.0 envsetup:docker /usr/src/app/apps/web
> > chmod +x ./env.sh && ./env.sh && mv ./env-config.js ./build/env-config.js
> 
> sh: ./env.sh: not found
>  ELIFECYCLE  Command failed.
>  ELIFECYCLE  Command failed with exit code 1.
> 
>  
> 
>  >  NX   Running target "start:static:build" failed
> ```
> 
> ### 🤖 Node Version
> _No response_
> 
> ### 📃 Provide any additional context for the Bug.
> _No response_
> 
> ### 👀 Have you spent some time to check if this bug has been raised before?
> * [x]  I checked and didn't find similar issue
> 
> ### 🏢 Have you read the Contributing Guidelines?
> * [x]  I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)
> 
> ### Are you willing to submit PR?
> None

I also tested with docker-compose, the same issue happened

**Comment by scopsy** [comment]
@oo00spy00oo could you please try the latest 0.10.1 version? Should be fixed there 🙏

**Comment by oo00spy00oo** [comment]
> @oo00spy00oo could you please try the latest 0.10.1 version? Should be fixed there 🙏

When will the next version be released? I cannot find the version number 0.10.1

**Comment by scopsy** [comment]
You can just fetch the new docker image with the `0.10.1` tag from here: https://github.com/novuhq/novu/pkgs/container/novu%2Fapi/61498832?tag=0.10.1
